contracts/VotingMachines/GenesisProtocol.sol

```diff
@@ -66,7 +68,7 @@ contract GenesisProtocol is IntVoteInterface,UniversalScheme,GenesisProtocolForm
     }
 
     event NewProposal(bytes32 indexed _proposalId, uint _numOfChoices, address _proposer, bytes32 _paramsHash);
-    event ExecuteProposal(bytes32 indexed _proposalId, uint _decision);
+    event ExecuteProposal(bytes32 indexed _proposalId, uint _decision,uint _totalReputation,ExecutionState _executionState);
```

Comment:
Should AbsoluteVote have a corresponding change, at least with regard to returning reputation?  And is it possible to keep this event signature consistent across voting machines?